docs/maintain/maintain-guides-how-to-chill.md

```diff
@@ -25,63 +23,35 @@ staking, take a look at the [accounts][] section in the general staking guide.
-can be the same validators if you prefer, or, a completely new set. Just be aware - your nominations
-will not persist across chills.
+When you chill after being a nominator, your nominations will be reset. This means that when you decide to start nominating again you will need to select validators to nominate once again. These can be the same validators if you prefer, or, a completely new set. Just be aware - your nominations will not persist across chills.
 
-Your nominator will remain bonded when it is chilled. When you are ready to nominate again, you will
-not need to go through the whole process of bonding again, rather, you will issue a new nominate
-call that specifies the new targets to nominate.
+Your nominator will remain bonded when it is chilled. When you are ready to nominate again, you will not need to go through the whole process of bonding again, rather, you will issue a new nominate call that specifies the new validators to nominate.
 
 ## Chilling as a Validator
 
-When you voluntarily chill after being a validator, your nominators will not automatically go away.
-As long as your nominators make no action, you will still have the nominations when you choose to
-become an active validator once again. However, if your nominators decide to nominate other
-validators then these nominations will take priority when the validator comes back. It may also be
-the case that your nominators change their entire nomination targets (all 16 of the allowed
-nominations). In this case, your nominators would need to explicitly specify your validator as a
-target when your validator comes back.
+When you voluntarily chill after being a validator, your nominators will remain.  As long as your nominators make no action, you will still have the nominations when you choose to become an active validator once again.  You bond however would not be listed as a nominable validator thus any nominators issuing new or revisions to existing nominations would not be able to select your bond.
+
+When you become an active validator you will also need to reset your validator preferences (commission, etc.). These can be configured as the same values that were set previously or something different.
 
-When you become an active validator you will also need to reset your validator preferences
-(commission, etc.). These can be configured as the same values that were set previously or something
-different.
+## Involuntary Chills
 
-### Involuntary Chills
+If a validator was unresponsive for an entire session the validator bond would be chilled in a process known as _involuntary chilling._ When a validator has been involuntarily chilled, it may restrict the validator from being selected in the next election depending on the session in which it was chilled (see considerations above).  A validator that was chilled may re-declare the intent to validate at any time, however, it is recommended that the validator attempt to determine the source of the chill before doing so.
```

Comment:
```suggestion
If a validator was unresponsive for an entire session, the validator bond would be chilled in a process known as _involuntary chilling._ When a validator has been involuntarily chilled, it may restrict the validator from being selected in the next election depending on the session in which it was chilled (see considerations above).  A chilled validator may re-declare the intent to validate at any time. However, it is recommended that the validator attempts to determine the source of the chill before doing so.
```